src/styles/tokens.scss

```diff
@@ -90,6 +90,7 @@ body {
   --color-fill-top-nav: var(--black0);
   --color-fill-top-banner: #9992;
   --color-fill-action: var(--brand5);
+  --color-fill-button: var(--brand5);
```

Comment:
This line is duplicated

Reply:
It's duplicated but `--color-fill-button` has different styles for dark mode. See here: https://www.figma.com/file/lOxAGGg5KXb6nwie7zXkz6/NJ---Design-System

Reply:
Will remove when I remove the pink button.